Session/Calls/CallVC.swift

```diff
@@ -464,9 +464,8 @@ final class CallVC: UIViewController, VideoPreviewDelegate, AVRoutePickerViewDel
         setUpViewHierarchy()
         setUpProfilePictureImage()
         
-        if shouldRestartCamera { cameraManager.prepare() }
```

Comment:
Just wondering will the camera work when users coming back to this screen from the minimised flowing window?